mlrun/api/crud/model_monitoring/helpers.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2018 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import json
+import typing
+
+import sqlalchemy.orm
+
+import mlrun.api
+import mlrun.common
+import mlrun.common.schemas.schedule
+import mlrun.errors
+
+
+def get_batching_interval_param(intervals_list: typing.List):
+    """Converting each value in the intervals list into a float number. None
```

Comment:
General comment on docstrings - don't use present participle ()"Convering", "Getting"), instead describe the function as "it": "Convert the batch interval..", "Retrieve access key from.." etc.